content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -186,19 +184,12 @@ In kubeadm terms, any certificate that would normally be signed by an on-disk CA
 
 ### Create certificate signing requests (CSR)
 
-You can pass in a directory with `--csr-dir` to output the CSRs to the specified location.
-If `--csr-dir` is not specified, the default certificate directory (`/etc/kubernetes/pki`) is used.
```

Comment:
i don't see this sentence duplicated:
> If `--csr-dir` is not specified, the default certificate directory (`/etc/kubernetes/pki`) is used.

better to move this sentence after:
> Both the CSR and the accompanying private key are given in the output; you can pass in a directory with `--csr-dir` to output the CSRs to the specified location.

Reply:
Agree, will do

Reply:
Done